clusterloader2/docs/create-new-test.md

```diff
@@ -0,0 +1,67 @@
+# Create a new performance test
+
+## Select and copy the baseline test job config
+
+A great majority of our tests lie at the [test-infra] repository. All the
+configs there are written as [ProwJobs]. There are two types of tests being used
+there:
+
+-   **periodic** which is a test that runs at given cron interval
+-   **presubmit** which may be used as a presubmit for GitHub pull requests
+
+And there are three scales at which those are run: 100, 500 and 5000 nodes. Once
+it's known what is to be achieved it's good to start with a test that is the
+closest to it. Here are a few good example starting points, all running
+[load-test][]:
+
+Presubmits:
+
+-   100 nodes: [pull-kubernetes-e2e-gce-100-performance]
+-   5000 nodes: [pull-kubernetes-e2e-gce-scale-performance-manual]
+
+Periodics:
+
+-   100 nodes: [ci-kubernetes-e2e-gci-gce-scalability]
+-   5000 nodes: [ci-kubernetes-e2e-gce-scale-performance]
+
+We propose to start with 100 node tests as they are less costly and much
+quicker, shortening debugging cycle and when those tests pass, switch to 5k node
+tests.
+
+## Copy and modify the selected baseline test
```

Comment:
@tosi3k You have a fresh experience with developing a ProwJob. Would it be ok for you to write a subsection here?

Reply:
Added:
``` markdown
### Developing a ProwJob

Before preparing the final version of the developed [ProwJob], it's good to run
a few tests manually on own resources to make sure that everything works fine. A
useful tool here is [Phaino] which allows submitting a ProwJob from local
workstation (no need to setup Prow). Please remember to make sure that the
ProwJob's `cluster` exists and set the GCP project via `--gcp-project` flag
(instead of `--gcp-project-type` boskos pool, unless Boskos is already set up).
```